src/functional/ffront/foast_passes/type_deduction.py

```diff
@@ -278,10 +278,9 @@ def visit_TupleTargetAssign(
                 if isinstance(index, tuple):
                     lower, upper = index
                     new_type = ct.TupleType(types=[t for t in values.type.types[lower:upper]])
-                    # TODO(samkellerhals): possibly change definition of Starred node to not need nested id attribute.
                     new_target = foast.Starred(
                         id=foast.DataSymbol(
-                            id=old_target.id.id, location=old_target.location, type=new_type  # type: ignore
+                            id=old_target.id.__str__(), location=old_target.location, type=new_type
```

Comment:
To be honest I am not sure whether this is acceptable, maybe best to get some feedback from @tehrengruber ?